### PR TITLE
Separate `nf` blocks in `evolve_pdfs`

### PR DESCRIPTION
**Pull request by giacomomagni**
Address #310. 

I believe 9d74a983 fixes the issue.
@andreab1997 if you are happy I'd like to merge this.

### CODE REVIEW

**Comment by andreab1997** [comment]
I can't know review it properly but at a first glance it seems correct. If you want to wait for my review I will probably do it tomorrow, otherwise I am also happy to merge it now

**Comment by giacomomagni** [comment]
> I can't know review it properly but at a first glance it seems correct. If you want to wait for my review I will probably do it tomorrow, otherwise I am also happy to merge it now

Thanks, let's way tomorrow, then.

File: src/ekobox/evol_pdf.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 """Tools to evolve actual PDFs."""
 import pathlib
+from collections import defaultdict
```
[the remark this answers is not in the document]
Comment (felixhekhorn) [reply src/ekobox/evol_pdf.py:3]: I'm wrong - sorry

**Review by andreab1997 — APPROVED** [review]
It looks ok to me. As @AleCandido said, I guess you already tried in some cases and it worked as expected, right?

**Comment by giacomomagni** [comment]
> It looks ok to me. As @AleCandido said, I guess you already tried in some cases and it worked as expected, right?

yes, it seems to work.

**Pull request merged**